src/Sulu/Bundle/MediaBundle/Resources/js/components/ImageFocusPoint/ImageFocusPointCell.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import {Icon} from 'sulu-admin-bundle/components';
+import type {Point} from './types';
+import imageFocusPointCellStyles from './imageFocusPointCell.scss';
+
+const ARROW_UP_ICON = 'arrow-up';
+
+type Props = {
+    size: number,
+    value: Point,
+    active: boolean,
+    onClick?: (value: Point) => void,
+    arrowDirection?: '-90deg' | '-45deg' | '0deg' | '45deg' | '90deg' | '125deg' | '180deg' | '225deg',
+};
+
+export default class ImageFocusPointCell extends React.PureComponent<Props> {
+    static defaultProps = {
+        active: false,
+        showArrow: true,
```

Comment:
Should be gone now.

Reply:
Seems like there is already a rule for that, at least if you are using prop-types: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/default-props-match-prop-types.md Maybe this also works with flow? Can you try?

Reply:
Tried that and it didn't do anything. Maybe it just works with `React.PropTypes`